### PR TITLE
feat: Include notification prop to display on Information Modal

### PR DESCRIPTION
**Pull request by vicchirino**
# Objective
Allow to display an inline notification on the Information Modal.

# Motivation and Context
On Performance we are updating modals when a manager is going to share a review. Currently we display an inline notification on the modal saying what is going to happen after the notification is shared. 

Include that notification as part of the `children` prop was generating that be stretched to the left. 

image

Added a `contentHeader ` prop into the modal to display the inline notification component between the title and modal content.

After a chat with Marc, we want the `InformationModal` when have an image on the right, flip the order of the actions buttons to have the primary action on the left side. For this include a `variant` prop on the `ModalFooter` component to avoid breaking changes.
Slack conversation: https://cultureamp.slack.com/archives/C0189KBPM4Y/p1614785352180800

# Screenshots (if appropriate)
image

# Checklist

- [x] If this contains visual changes, has it been reviewed by a designer?
- [ ] If this introduces either a new component or a breaking change to an existing component, has it been reviewed by an advocate? (Ask in #prod_design_systems)
- [ ] I have considered likely risks of these changes and got someone else to QA as appropriate
- [x] I have or will communicate these changes to the front end practice

### CODE REVIEW

File: draft-packages/modal/docs/Modal.stories.tsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { title } from "process"
```
Comment (ActuallyACat) [inline draft-packages/modal/docs/Modal.stories.tsx:1]: This import looks like an accident.

File: draft-packages/modal/docs/Modal.stories.tsx [hunk 2]
```diff
@@ -705,6 +710,74 @@ InformationModalWithImage.story = {
...
+            </Heading>
+          </ModalAccessibleDescription>
+          <Padding />
+          <Paragraph variant="body">
+            • What did this person achieve over the last 6 months? What specific
+            impact did they have on the team’s goals?
+          </Paragraph>
+          <Padding />
+          <Paragraph variant="body">
+            • What progress have they made on their learning anddevelopment
```
Comment (ActuallyACat) [inline draft-packages/modal/docs/Modal.stories.tsx:758]: /nit `anddevelopment` => `and development`

File: draft-packages/modal/KaizenDraft/Modal/Presets/InformationModal.tsx [hunk 3]
```diff
@@ -25,6 +32,7 @@ export type InformationModalProps = Readonly<
     renderBackground?: () => React.ReactNode
     image?: React.ReactNode
     children: React.ReactNode
+    notification?: InlineNotificationProps
```
Comment (ActuallyACat) [inline draft-packages/modal/KaizenDraft/Modal/Presets/InformationModal.tsx:35]: Something like `contentHeader?: children` would be preferred so it's a little more generic.

Comment (vicchirino) [reply draft-packages/modal/KaizenDraft/Modal/Presets/InformationModal.tsx:35]: yes will do that! Thanks 🙏

**Review by ActuallyACat — APPROVED** [review]
LGTM - nice one @vicchirino

**Pull request merged**